.github/workflows/ci_cd.yml

```diff
@@ -45,6 +45,19 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
+      - name: "Setup python"
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+
+      - name: "Checkout repo"
+        uses: actions/checkout@v4
+
+      - name: "Install dependencies"
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[changelog]
```

Comment:
I see... These are required for Sphinx design, right?

Reply:
yes, Incase if there is no sphinx design in their doc dependency